test/types/add.ts

```diff
@@ -0,0 +1,9 @@
+import { expectType, expectError} from 'tsd';
+import * as R from '../../es/index';
+
+expectType<number>(R.add(2, 3)); // => 5
```

Comment:
it's generally because test output is copied from the examples from the docs.
that said, if it gets out of sync then i feel like that would mean something's gone *very* wrong - since that would be a breaking change... more importantly it'd mean the current definition is wrong and i feel like most of those would have already been caught